drivers/hopwa_caper/app/views/hopwa_caper/cells/show.haml

```diff
@@ -0,0 +1,44 @@
+= content_for :crumbs do
+  = link_to path_for_question_result(@question, report: @report) do
+    = "« #{report_short_name} #{@question} Results"
+- content_for :title do
+  = "#{report_short_name} #{@question} #{@cell}"
+.d-flex.mb-2
+  %h1= content_for :title
+  .ml-auto
+    = link_to(link_params.merge({format: :xlsx}),{class: 'btn btn-secondary'}) do
+      %i.icon-file-excel
+      Download
+
+
+
+- if @clients.blank?
+  .none-found No records found.
+- else
+  .card
```

Comment:
It's probably worth adding the live search logic here, see `drivers/core_demographics_report/app/views/core_demographics_report/warehouse_reports/shared/_details.haml` for an example.  (Two bits, the search form and the JS)